addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/WelcomeWebHookServlet.java

```diff
@@ -38,29 +32,35 @@
  *
  * @author Gaël L'hopital - Initial contribution
  */
-@Component(service = HttpServlet.class, configurationPolicy = ConfigurationPolicy.OPTIONAL, immediate = true)
 public class WelcomeWebHookServlet extends HttpServlet {
     private static final long serialVersionUID = 1288539782077957954L;
-    public static final String PATH = "/netatmo/camera";
+    private static final String PATH = "/netatmo/%s/camera";
     private static final String APPLICATION_JSON = "application/json";
     private static final String CHARSET = "utf-8";
     private final Gson gson = new Gson();
 
     private final Logger logger = LoggerFactory.getLogger(WelcomeWebHookServlet.class);
 
     private HttpService httpService;
-    private NetatmoHandlerFactory netatmoHandlerFactory;
+    private NetatmoBridgeHandler bridgeHandler;
+    private String path;
+
+    public WelcomeWebHookServlet(@NonNull HttpService httpService, String id) {
+        super();
```

Comment:
This should normally not be needed, calling super in the begining of the constructor is the default behaviour.

Reply:
Ok, will suppress it.